### ISSUE TITLE
Wrong foldername when adding from Trakt

### ISSUE THREAD

**Issue by selthius** [SickRage/sickrage-issues#256]
When addin a how from a Trakt list, the title used by trakt is chosen as the folder name. For instance, Doctor Who (2005) is named just Doctow Who and Battlestar Galactica (2003( is named just Battlestar Galactica.

Because of this, some applications will have problems identifying the correct tv show. In addition, if the correct show for that folder is later added, an error occurs.

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist)

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by labrys** [comment]
@Thraxis is this still pending?

**Comment by Thraxis** [comment]
Yes at the moment.  RL took a bit of a forefront during the holidays.  There are a couple ways this can be addressed, just fixing it for shows added from lists is one way.  The other is to stop the system from creating folders for shows first and waiting til it has information from the Indexer.   this would also prevent the cases where you get orphaned folders because TVDB doesn't have any season / episode information yet

**Comment by labrys** [comment]
@Thraxis I figured RL and holidays; just trying to not let issues fall by the wayside.. not rushing you ofc.

**Comment by p0psicles** [comment]
@Thraxis you think can continue working on this? Or do you want to tag out? As i'm currently into the lists, i can at least look at option 1. And if that's resolved we can always take a look at option 2?

**Comment by p0psicles** [comment]
Okay, I started on this as I needed a solution for adding anidb shows. That has the same problem as for the trakt lists or for the imdb lists.

https://github.com/SickRage/SickRage/pull/810

**Comment by OmgImAlexis** [comment]
@p0psicles you should really be using separate branches next time.

**Comment by p0psicles** [comment]
@OmgImAlexis i am using a separate branch. But I needed to solve this issue for my own anidb list functionality. Can't help solving another issue at the same time.

**Comment by Thraxis** [comment]
@p0psicles - I was going to try and get to this on the weekend.  Just been alot of things piling up with RL work.   Briefly looking at what you did that is what I was looking at doing as well.

Honestly you have been mostly covering the items I wanted to work on but haven't had a chance.

My only other personal project I hadn't posted about was to work on implementing an "default anime profile" to set the defaults for adding anime shows.  and then implement a lookup using the lists from https://github.com/ScudLee/anime-lists to see if a TVDBID was an anime or not and automatically choosing the anime default when adding.

**Comment by p0psicles** [comment]
Well there will be enough work left for the both of us!
I'll be adding the anidb latest anime list this weekend. One of the PR's I just did, was to fix the trakt directory name (using your option 2) solution. Maybe you can verifty this for me?

@OmgImAlexis i'm doing the PR's in smaller chunks now against development. That way you can also get around to reviewing them much sooner!
